src/main/java/de/tum/in/www1/artemis/repository/ResultRepository.java

```diff
@@ -306,12 +348,13 @@ default DueDateStat countNumberOfFinishedAssessmentsForExercise(Long exerciseId,
     /**
      * Given a courseId, return the number of assessments for that course that have been completed (e.g. no draft!)
      *
+     * !! this is very slow - 3787 ms
```

Comment:
What bout this?

Reply:
that's a todo for a followup :)

Reply:
Maybe add a TODO then so we know there is still work to be done